Retarus/retarus_modules/retarus_events_forwarder.py

```diff
@@ -0,0 +1,116 @@
+import uuid
+from datetime import datetime, timezone
+from queue import Empty, SimpleQueue
+from threading import Event, Thread
+
+import dateutil.parser
+from kafka import KafkaProducer
+from kafka.errors import KafkaTimeoutError
+from sekoia_automation.connector import Connector
+
+from retarus_modules.metrics import OUTCOMING_EVENTS
+
+
+class RetararusEventsForwarder(Thread):
```

Comment:
Events should be forwarded to Sekoia.io thanks to the `Connector.push_events_to_intakes().`

Reply:
I think it's more in line with what's expected now